### PR TITLE
Add GUI Buttons To Example Scripts (All Help Commands)

### PR DESCRIPTION
**Pull request by OliverGlandberger**
_**'EXTRA' : This PR also fixes/adds the following**_
- The 'qtm' module was not accessible from both 'startup.py' files. This PR adds "import qtm" to both.
- Improved consistency of **@staticmethod** usage in 'custom_menu_bar' class

---

The current accessibility to the help functionality is less ideal, especially for new users of the scripting API. This PR adds GUI buttons for all the available "help" functions that can be called in the Python terminal.

**There is one menu for the "root-level" help messages (_"Main Topics"_):**

image

**There is one menu for the "module-specific" & "function-specific" help messages:**

image

**Pressing the button shown in the example image above prints the following into the Python terminal:**

image

### CODE REVIEW

File: helpers/menu_tools.py [hunk 1]
```diff
@@ -145,6 +145,24 @@ def add_menu_item(menu_id, button_text, command_name, index=None):
     return True
 
 
+def get_menu_item_index(menu_id, name):
```
Comment (bahlborn) [inline helpers/menu_tools.py]: comment: this looks unused

File: demo_scripts/Classes/custom_menu_bar_class.py [hunk 2]
```diff
@@ -173,6 +175,77 @@ def _add_commands_and_buttons_open_files_folders(open_folders_menu_id, open_scri
                     if curr_file.rsplit(".", 1)[1] == "py" and curr_file != "__init__.py":
                         add_command("open_script_" + curr_file, lambda local_var=folder_path + "\\" + curr_file: (subprocess.Popen(["notepad.exe", local_var], shell=True)))
                         add_menu_item(open_scripts_menu_id, curr_file, "open_script_" + curr_file)
+
+    @staticmethod
+    def _add_help_root_menu_items(menu_id):
+        for curr_command in qtm.gui.get_commands():
+            if "help_root_" in curr_command:
```
Comment (bahlborn) [inline demo_scripts/Classes/custom_menu_bar_class.py]: suggestion: use a constant for "help_root_"

File: helpers/menu_tools.py (hunk 1 above)
Comment (OliverGlandberger) [reply helpers/menu_tools.py]: Good call. I never ended up using this function, and the function below it was originally used, but one of the later commits to this PR removes all usages of that function, so both functions are unused... Removed both!

File: demo_scripts/Classes/custom_menu_bar_class.py (hunk 2 above)
Comment (OliverGlandberger) [reply demo_scripts/Classes/custom_menu_bar_class.py]: Renamed to "root_help_" and also made it a constant. I also moved the "leaking" menu-item creation code that was previously placed in the 'startup' file to the 'custom_menu_bar', which is a more correct location.

**Review by ulvs — APPROVED** [review]

**Pull request merged**